_translator-files/po/de/Command-Line-Options.po

```diff
@@ -41,17 +42,15 @@ msgstr "Jamulus - Befehlszeilen Optionen"
 
 #. type: Title #
 #: ../wiki/en/Command-Line-Options.md:8
-#, fuzzy, no-wrap
-#| msgid "# Command Line Options"
+#, no-wrap
 msgid "Command Line Options"
-msgstr "# Befehlszeilen Optionen"
+msgstr "Befehlszeilenoptionen"
 
 #. type: Title ##
 #: ../wiki/en/Command-Line-Options.md:10
-#, fuzzy, no-wrap
-#| msgid "## Shared commands"
+#, no-wrap
 msgid "Shared commands"
-msgstr "## Befehle für Client und Server"
+msgstr "Gemeinsame Befehle"
```

Comment:
I'm not that fond of "Shared commands" as it's not understandable out of context.